build.sbt

```diff
@@ -212,23 +175,18 @@ lazy val docs = project
         .withRepository(uri("https://github.com/BlueBrain/nexus"))
         .withSocial(
           uri("https://github.com/BlueBrain"),
-          uri("https://gitter.im/BlueBrain/nexus")
+          uri("https://github.com/BlueBrain/nexus/discussion")
```

Comment:
the url should be https://github.com/BlueBrain/nexus/discussions
notice the trailing s